integration-tests/src/test/java/com/vrg/rapid/integration/AbstractMultiJVMTest.java

```diff
@@ -193,20 +196,20 @@ RapidNodeRunner killOnExit() {
         boolean killNode() throws IOException {
 
             // Interval to wait after shutdown retry
-            Long SHUTDOWN_RETRY_WAIT = 500L;
+            final Long shutdownRetryWait = 500L;
             // Number of retries to kill node before giving up.
-            int SHUTDOWN_RETRIES = 10;
+            final int shutdownRetries = 10;
```

Comment:
If these are constants, why not have them as private static final constants for the class? I would name SHUTDOWN_RETRY_WAIT something like SHUTDOWN_RETRY_WAIT_MS indicating the time unit in the name of the constant.